nestkernel/mpi_manager.cpp

```diff
@@ -713,6 +713,30 @@ nest::MPIManager::grng_synchrony( unsigned long process_rnd_number )
   return true;
 }
 
+// wfr_synchrony: called at the beginning of each simulate
+bool
+nest::MPIManager::wfr_synchrony( bool any_node_uses_wfr )
```

Comment:
@heplesser You understand it right and I agree. I made an attempt to improve the naming

1. `any` is indeed very short. I chose `any_true` instead. This indicates that this is an operation with boolean parameters.
2. I took a look at the MPI standard but did not find anything like this.